.gitignore

```diff
@@ -10,6 +10,7 @@ src/StaticAnalysisResultsInterchangeFormat/Autogenerated/
 *.user
 *.userosscache
 *.sln.docstates
+*.opendb
```

Comment:
I guess Update 2 is emitting this file (something.VC.opendb), so I added it to the .ignore list.